src/server/controllers/movie.js

```diff
@@ -1,31 +1,66 @@
-const jwt = require('jsonwebtoken');
-const { PrismaClient } = require('@prisma/client');
+const jwt = require("jsonwebtoken");
+const { PrismaClient } = require("@prisma/client");
 const prisma = new PrismaClient();
 
-const jwtSecret = 'mysecret';
+const jwtSecret = "mysecret";
```

Comment:
Move this to an env var and load it from there using `process.env`

Reply:
Stored in .env file ✔️